### ISSUE TITLE
The CRF filed is missing in videocombine node

### ISSUE THREAD

**Issue by maxbizz** [Kosinkadink/ComfyUI-VideoHelperSuite#339]
In my comfyui ,i cant find the CRF field in the videocombine node. I dont know whats gone wrong here. I have updated the nodes to the latest already

**Comment by AustinMroz** [comment]
crf is only displayed if you have selected a format which that provides crf as an option (`video/h264-mp4`, `video/h265-mp4`, `video/webm`, and `video/av1-webm`)

**Comment by maxbizz** [comment]
I can confirm that this is not working. I have checked all the formats(video/h264-mp4, video/h265-mp4, video/webm). CRF field is still hidden

**Comment by maxbizz** [comment]
Got this resolved. the h264 json file from another node is getting used instead of the json files inside videoformat folder of VHS node.

**Comment by FlareP1** [comment]
Thanks for this, I had the same problem, was driving me crazy.  In my case the 'ComfyUI-tbox' node was overriding the settings.  Is there anyway to ensure that the settings are only loaded from the ComfyUI-VideoHelperSuite directories ?

**Comment by maxbizz** [comment]
Yes, tthat's the same node creating the issue in my case as well..I just uninstalled that node.

**Comment by AustinMroz** [comment]
A conflicting node with have been in my next couple guesses of things to check, but I'm glad to hear you've figured it out.

ComfyUI-tbox has stolen code from VHS and violated the GPLv3 license it is under. It overrides the `VHS_video_formats` location, which is clearly namespaced to VHS, and leaves me with limited options.

I could no longer register video_formats as a folder path. This would remove the ability for users to specify an additional format location outside of VHS, but I'm not sure anyone has made use of such functionality.

**Comment by HIROKIxYUKI** [comment]
THANK YOU! this was driving me crazy, uninstalling ComfyUI-tbox worked.
